samples/CommunityToolkit.Maui.Sample/Pages/Views/Popup/StylePopupPage.xaml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<pages:BasePage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:pages="clr-namespace:CommunityToolkit.Maui.Sample.Pages"
+             xmlns:viewModels="clr-namespace:CommunityToolkit.Maui.Sample.ViewModels.Views"
+             x:TypeArguments="viewModels:StylePopupViewModel"
+             x:DataType="viewModels:StylePopupViewModel"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Views.StylePopupPage"
+             Title="StylePopupPage">
+
+    <ScrollView Orientation="Vertical">
+        <StackLayout Orientation="Vertical" Spacing="5">
+            <Label Text="Popup can use Styles"
+                   HorizontalOptions="Center"
+                   HorizontalTextAlignment="Center"
+                   VerticalTextAlignment="Start"/>
+            
+            <Button
+                Text="Implicit Style"
+                BackgroundColor="Blue"
+                TextColor="White"
+                Command="{Binding DisplayPopup1Command}"/>
```

Comment:
It might be nicer to name these commands so that they provide clearer context to what they are doing, e.g. `DisplayImplicitStylePopupCommand`, etc.

Reply:
I agree with your opinion. Change the name of the command to something easier to understand.